examples/csc/plot_lfp_data.py

```diff
@@ -14,13 +14,28 @@
 ###############################################################################
 # First, let us fetch the data (~14 MB)
 import os
-from mne.utils import _fetch_file
+from mne.datasets import fetch_dataset
+from mne.utils import get_config
 
 url = ('https://github.com/hitziger/AWL/raw/master/Experiments/data/'
        'LFP_data_contiguous_1250_Hz.mat')
-fname = './LFP_data_contiguous_1250_Hz.mat'
-if not os.path.exists(fname):
-    _fetch_file(url, fname)
+
+folder_name = "LFP"
+archive_name = "LFP_data_contiguous_1250_Hz.mat"
+
+fetch_dataset(
+    {"dataset_name": "LFP_data",
+     "url": url,
+     "archive_name": archive_name,
+     "folder_name": folder_name,
+     "hash": None
+     },
+    path=None,
+    force_update=False
+)
```

Comment:
`fetch_dataset` returns the path of the data so you could directly use this instead of `get_config` no?
https://github.com/mne-tools/mne-python/blob/main/mne/datasets/_fetch.py#L89

Reply:
Yes, you're right; though it returns the directory path rather than file path. I do not know if it is a bug on mne side.

I've changed it.

Reply:
This one is not a bug :) Dataset can have multiple files so it makes sense to have the folder containing all the files returned.
Thx for fixing this!